docs/integrations/data-ingestion/clickpipes/mongodb/pause_and_resume.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Pausing and Resuming a MongoDB ClickPipe'
+description: 'Pausing and Resuming a MongoDB ClickPipe'
+sidebar_label: 'Pause Table'
+slug: /integrations/clickpipes/mongodb/pause_and_resume
+---
+
+import Image from '@theme/IdealImage';
+import pause_button from '@site/static/images/integrations/data-ingestion/clickpipes/postgres/pause_button.png'
+import pause_dialog from '@site/static/images/integrations/data-ingestion/clickpipes/postgres/pause_dialog.png'
+import pause_status from '@site/static/images/integrations/data-ingestion/clickpipes/postgres/pause_status.png'
+import resume_button from '@site/static/images/integrations/data-ingestion/clickpipes/postgres/resume_button.png'
+import resume_dialog from '@site/static/images/integrations/data-ingestion/clickpipes/postgres/resume_dialog.png'
+
+There are scenarios where it would be useful to pause a MongoDB ClickPipe. For example, you may want to run some analytics on existing data in a static state. Or, you might be performing upgrades on MongoDB. Here is how you can pause and resume a MongoDB ClickPipe.
+
+## Steps to pause a MongoDB ClickPipe {#pause-clickpipe-steps}
+
+1. In the Data Sources tab, click on the MongoDB ClickPipe you wish to pause.
+2. Head over to the **Settings** tab.
+3. Click on the **Pause** button.
+
+<Image img={pause_button} border size="md"/>
+
+4. A dialog box should appear for confirmation. Click on Pause again.
+
+<Image img={pause_dialog} border size="md"/>
+
+4. Head over to the **Metrics** tab.
+5. In around 5 seconds (and also on page refresh), the status of the pipe should be **Paused**.
```

Comment:
> (and also on page refresh)

For PG pipes the status updates automatically (even after 40 seconds) and the page refresh doesn't help. Is it different here?

If same, the there are a couple more mentions of page refresh

Reply:
good catch, re: `40 seconds` i think think is a regression in recent behavior. 

I will remove the refreshes part -- it is not different here.